test/CMakeLists.txt

```diff
@@ -17,11 +17,13 @@ if (NOT GTest_FOUND)
       message(STATUS "Attempting to apply Debian fix for googletest")
```

Comment:
Something is broken here. The code currently does not build on Debian 8 or 9 as the `FindGTest` module could not find the libraries.